kernel/src/model/common.rs

```diff
@@ -11,10 +11,10 @@ pub enum OutputFormat {
 impl fmt::Display for OutputFormat {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         let s = match self {
-            OutputFormat::Json => "JSON",
-            OutputFormat::Sarif => "SARIF",
+            Self::Json => "JSON",
```

Comment:
It's usually a good practice to use `Self` in these scenarios. See  [this](https://rust-lang.github.io/rust-clippy/master/index.html#/use_self).